integration-tests/e2e-runtime-only-backend/lib.rs

```diff
@@ -121,7 +121,7 @@ pub mod flipper {
 
             // when
             let call_data = vec![
-                Value::from_bytes(&contract.account_id),
+                Value::unnamed_variant("Id", [Value::from_bytes(contract.account_id)]),
```

Comment:
I think it may have sense to understand why and when it has changed.
I also noticed that the drink client returns metadata in version V14. Shouldn't it be V15?

Reply:
This is the issue that we should resolve before the final release. For now, I'm just gonna push the release gracefully.